src/applications/hca/config/chapters/insuranceInformation/general.js

```diff
@@ -0,0 +1,72 @@
+import { merge, get } from 'lodash/fp';
+import fullSchemaHca from 'vets-json-schema/dist/10-10EZ-schema.json';
+
+import InsuranceProviderView from '../../../components/InsuranceProviderView';
+
+const { provider } = fullSchemaHca.definitions;
+const { isCoveredByHealthInsurance } = fullSchemaHca.properties;
+
+export default {
+  uiSchema: {
+    'ui:title': 'Other coverage',
+    isCoveredByHealthInsurance: {
+      'ui:title':
+        'Are you covered by health insurance? (Including coverage through a spouse or another person)',
+      'ui:widget': 'yesNo',
+    },
+    providers: {
+      'ui:options': {
+        itemName: 'Insurance Policy',
+        expandUnder: 'isCoveredByHealthInsurance',
+        viewField: InsuranceProviderView,
+      },
+      'ui:errorMessages': {
+        minItems: 'You need to at least one provider.',
```

Comment:
Language is weird here... Use `Please add at least one provider?` maybe?

Reply:
Yess sir made a [ticket](https://app.zenhub.com/workspaces/vsa---caregiver-5fff0cfd1462b6000e320fc7/issues/department-of-veterans-affairs/va.gov-team/19435) to go over all content particularly validation message here as per your feedback

Also made this [ticket](https://app.zenhub.com/workspaces/vsa---caregiver-5fff0cfd1462b6000e320fc7/issues/department-of-veterans-affairs/va.gov-team/19438) for checkbox groups, @Mottie